packages/test/test-drivers/src/odspDriverApi.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    OdspDocumentServiceFactory,
+    createOdspCreateContainerRequest,
+    createOdspUrl,
+    OdspDriverUrlResolver,
+} from "@fluidframework/odsp-driver";
+import { pkgVersion } from "./packageVersion";
+
+export const OdspDriverApi = {
+    version: pkgVersion,
+    OdspDocumentServiceFactory,
+    OdspDriverUrlResolver,
+    createOdspCreateContainerRequest,
+    createOdspUrl,                          // REVIEW: does this need to be back compat?
```

Comment:
@curtisman is there a follow-up here? Open an issue perhaps?

Reply:
No follow-up here. mostly just a comment that I am not sure it needs to be tested for back compat or not.  I am just leaving it in for now, and decision can be made in the future as scenario warrants its removal.